src/main/java/frc/robot/subsystems/LEDManager.java

```diff
@@ -26,34 +29,69 @@
  */
 public class LEDManager {
 
+    private static LEDManager instance;
+
     // Hardware
     private AddressableLED ledStrip;
     private AddressableLEDBuffer ledBuffer;
 
     /**
-     * Creates a new LEDManager object using the provided PWM port number.
+     * returns the singleton object of the LED Manager
+     * @return the singleton object
+     */
+    public static LEDManager getInstance() {
+        if (instance == null) {
+            throw new IllegalStateException("LEDManager has not been initialized. Call initialize() first.");
+        } else {
+            return instance;
+        }
```

Comment:
A style point most code linters will point out; in the case of an `if` that throws or returns, you don't need an else. 
```suggestion
        if (instance == null) {
            throw new IllegalStateException("LEDManager has not been initialized. Call initialize() first.");
        }

        return instance;
```